.github/workflows/devcontainer.yml

```diff
@@ -0,0 +1,24 @@
+name: Dev Containers
+
+on:
+  workflow_dispatch:
+  push:
+    branches: ["main"]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: ["main"]
+
+jobs:
+  build:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Build dev containers
```

Comment:
What are we doing here, building a docker container for each dev-container env? What is the advantage?